consul-lambda-registrator/structs/service.go

```diff
@@ -5,6 +5,8 @@ import (
 	"net/url"
 	"strconv"
 	"strings"
+
+	"github.com/hashicorp/consul/api"
```

Comment:
I believe this gets pulled into the extension too. At some point, it might be interesting to see how much this dependency increases the size of the Lambda extension.